src/ripple/net/impl/RPCCall.cpp

```diff
@@ -202,6 +218,25 @@ class RPCParser
         return v;
     }
 
+    Json::Value parseManifest (Json::Value const& jvParams)
+    {
+        if (1 == jvParams.size ())
+        {
+          Json::Value     jvRequest (Json::objectValue);
+
+          std::string const strPk = jvParams[0u].asString ();
+          if (!validPublicKey (strPk))
+            return rpcError (rpcPUBLIC_MALFORMED);
+
+          jvRequest[jss::public_key] = strPk;
```

Comment:
If `validPublicKey` returned `boost::optional<PublicKey>` then we could return the public key in "canonical" format (whether base58 or hex) instead of whichever format the user provided it for us.

Reply:
Done